client/src/modules/tags/modal/createUpdate.js

```diff
@@ -2,12 +2,18 @@ angular.module('bhima.controllers')
   .controller('TagsModalController', TagsModalController);
 
 TagsModalController.$inject = [
-  'data', '$state', 'TagsService', 'NotifyService',
-  '$uibModalInstance', '$rootScope',
+  'data', 'TagService', 'NotifyService',
+  '$uibModalInstance', '$rootScope', 'ColorService',
 ];
 
-function TagsModalController(data, $state, TagsService, Notify, Instance, $rootScope) {
+function TagsModalController(
+  data, TagsService, Notify, Instance, $rootScope, Colors,
+) {
   const vm = this;
+  vm.colors = Colors.list.map(e => {
+    e.iconColor = { color : e.value, 'font-size' : '14px' };
```

Comment:
Since this is more than a `iconColor` (it has the font size in it), it might make sense to call it `iconStyle`.